contrib/gcppubsub/config/gcppubsub.yaml

```diff
@@ -161,6 +161,14 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
```

Comment:
While doing the refactor and testing it, I found that this was missing. Already added as part of another PR, and has been merged to master...